nirc_ehr/resources/queries/study/clinical_observationsSummary.sql

```diff
@@ -0,0 +1,21 @@
+SELECT
+    Id,
+    date,
+    category,
+    area,
+    observation,
+    remark,
+    performedBy,
+    scheduledDate,
+    QCState
+FROM clinical_observations
+GROUP BY
```

Comment:
Does it need an aggregate column for the group by?

Reply:
No this is just to not show duplicates. No aggregate necessary